qemu/disklabel

```diff
@@ -0,0 +1,10 @@
+/            2G
```

Comment:
A readme in the `qemu` dir that explains what these different filer are used for would be really helpful.